Manifests/ManifestsApple/com.apple.loginwindow.plist

```diff
@@ -770,6 +771,35 @@ Availability: Available in macOS 10.13 and later.</string>
 			<string>Show input menu in login window</string>
 			<key>pfm_type</key>
 			<string>boolean</string>
+		</dict>
+				<dict>
```

Comment:
Can you please fix the `<dict>` alignment here?

Reply:
Fixed in the latest commit.